it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CaptureCompanyNumberControllerISpec.scala

```diff
@@ -43,7 +43,8 @@ class CaptureCompanyNumberControllerISpec extends ComponentSpecHelper
         optServiceName = None,
         deskProServiceId = testDeskProServiceId,
         signOutUrl = testSignOutUrl,
-        businessEntity = LimitedCompany
+        businessEntity = LimitedCompany,
+        businessVerificationCheck = testBusinessVerificationCheck
```

Comment:
I'm not a fan of this being passed in, I think it would read better just as true here rather than a test variable